### PR TITLE
Update ns-Microsoft.AspNetCore.xml

### PR DESCRIPTION
**Pull request by pranavkm**

### CODE REVIEW

File: aspnet-core/xml/ns-Microsoft.AspNetCore.xml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <Namespace Name="Microsoft.AspNetCore">
   <Docs>
-    <summary>To be added.</summary>
-    <remarks>To be added.</remarks>
+    <summary>Provides types that that enable getting started with building an ASP.NET app with opinionated defaults.</summary>
+    <remarks>For for more information, see https://docs.microsoft.com/en-us/aspnet/core/fundamentals/host/web-host.</remarks>
```
Comment (Rick-Anderson) [inline aspnet-core/xml/ns-Microsoft.AspNetCore.xml:4]:
```suggestion
    <remarks>For for more information, see https://docs.microsoft.com/aspnet/core/fundamentals/host/web-host.</remarks>
```

**Review by Rick-Anderson — APPROVED** [review]
After removing culture /en-en
So it creates the correct link in all locals.

File: aspnet-core/xml/ns-Microsoft.AspNetCore.xml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 <Namespace Name="Microsoft.AspNetCore">
   <Docs>
-    <summary>To be added.</summary>
-    <remarks>To be added.</remarks>
+    <summary>Provides types that that enable getting started with building an ASP.NET app with opinionated defaults.</summary>
```
Comment (JamesNK) [inline aspnet-core/xml/ns-Microsoft.AspNetCore.xml:3]:
```suggestion
    <summary>Provides types that enable getting started with building an ASP.NET app with opinionated defaults.</summary>
```

File: aspnet-core/xml/ns-Microsoft.AspNetCore.xml [hunk 3]
```diff
@@ -1,6 +1,6 @@
 <Namespace Name="Microsoft.AspNetCore">
   <Docs>
-    <summary>To be added.</summary>
-    <remarks>To be added.</remarks>
+    <summary>Provides types that enable getting started with building an ASP.NET app with opinionated defaults.</summary>
+    <remarks>For for more information, see https://docs.microsoft.com/aspnet/core/fundamentals/host/web-host.</remarks>
```
Comment (JamesNK) [inline aspnet-core/xml/ns-Microsoft.AspNetCore.xml:4]:
```suggestion
    <remarks>For for more information, see <see href="https://docs.microsoft.com/aspnet/core/fundamentals/host/web-host">ASP.NET Core Web Host</see>.</remarks>
```

**Review by JamesNK — APPROVED** [review]

**Pull request merged**